src/structures/api/SummonerChallenge.ts

```diff
@@ -131,6 +131,14 @@ export class ChallengePreferences {
    * The IDs of the challenges this summoner has put on display.
    */
   readonly displayedChallengeIds: number[];
+  /**
+   * Type of the border the summoner has currently selected. "1" is level border, "2" is highest rank border.
+   */
+  readonly crestBorder: string;
+  /**
+   * Level of the border the summoner has currently selected.
```

Comment:
Would be clearer to use - "The level at which summoner unlocked currently equipped level border."